sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +33,160 @@ TypeScript users need to have Node type definitions installed:
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyVaultBackupClient } from "@azure/keyvault-admin";
+
+const credentials = new DefaultAzureCredential();
+
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const client = new KeyVaultBackupClient(vaultUrl, credentials);
+```
 
 ## Key concepts
 
-## Authenticating with Azure Active Directory
+### KeyVaultRoleDefinition
+
+A Role Definition is a collection of permissions. A role definition defines the operations that can be performed, such as read, write, 
+and delete. It can also define the operations that are excluded from allowed operations.
+
+Role definitions can be listed and specified as part of a Role Assignment.
+
+### KeyVaultRoleAssignment. 
+
+A Role Assignment is the association of a Role Definition to a service principal. They can be created, listed, fetched individually, and deleted.
+
+### KeyVaultAccessControlClient
+
+A [KeyVaultAccessControlClient][src-rbac-client] provides both synchronous and asynchronous operations allowing for management of Role Definitions (instances of `KeyVaultRoleDefinition`) and Role Assignments (instances of `KeyVaultRoleAssignment`).
+
+### KeyVaultBackupClient
+
+A [KeyVaultBackupClient][src-backup-client] provides both synchronous and asynchronous operations for performing full key backups, full key restores, and selective key restores.
+
+### Long Running Operations
+
+The Azure SDK for TypeScript and JavaScript offers a common abstraction for long running operations through the package [core-lro][core-lro]. In particular, the operations done by the [KeyVaultBackupClient][src-backup-client] may take as much time as needed by the Azure resources, requiring a client layer to keep track, serialize and resume the operations through the lifecycle of the programs that wait for them to finish.
+
+The [KeyVaultBackupClient][src-backup-client] offers three methods that execute Long Running Operations:
+
+- `beginBackup`, starts generating a backup of an Azure Key Vault on the specified Storage Blob account.
+- `beginRestore`, starts restoring all key materials using the SAS token pointing to a previously stored Azure Blob storage backup folder.
+- `beginSelectiveRestore`, starts restoring all key versions of a given key using user supplied SAS token pointing to a previously stored Azure Blob storage backup folder.
+
+All of the Long Running Operation methods return a a Long Running Operation poller that allows you to wait indefinitely until the Key Vault selective restore is complete. More information is available on the examples below.
 
 ## Examples
 
+The following section provides several code snippets using the `client` created above for either [access control](#create-keyvaultaccesscontrolclient) or [backup](#create-KeyVaultBackupClient) clients, covering some of the most common Azure Key Vault access control related tasks:
```

Comment:
> covering some of the most common Azure Key Vault access control related tasks

Since both access control and backup clients are referred to in this section, it is odd that we say that the section covers the most common "access control related tasks"

> using the `client` created above

The reference to "client created above" is not needed and forces one who lands on this section directly to then scroll to another section. Since the samples being linked from here have the client creation code, we do not need to link to client creation ourselves.

```suggestion
The following section provides several code snippets using the `KeyVaultAccessControlClient` for access control tasks and `KeyVaultBackupClient` for backup/restore tasks.
```

Reply:
I like this! thank you 🌞

Reply:
Here's what I'm placing in the examples:

> ## Examples
> 
> We have provided readmes with the detailed steps to run our samples, both for [JavaScript](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/javascript/README.md) and [TypeScript](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-admin/samples/typescript/README.md). Direct links to the specific samples follow:
>
> ...

Feedback appreciated!